Makefile

```diff
@@ -47,4 +47,18 @@ clean: destroy-box remove-tmp
 
 # helper commands
 update-box:
-	@SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} vagrant box update || (echo '\n\nIf you get an SSL error you might be behind a transparent proxy. \nMore info https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/README.md#if-you-are-behind-a-transparent-proxy\n\n' && exit 2)
\ No newline at end of file
+	@SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} vagrant box update || (echo '\n\nIf you get an SSL error you might be behind a transparent proxy. \nMore info https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/README.md#if-you-are-behind-a-transparent-proxy\n\n' && exit 2)
+
+proxy-redash:
+	consul connect proxy -service=proxy-to-redash -upstream=redash-redash-server-service:7070 -log-level=TRACE
+
+proxy-presto:
+	consul connect proxy -service=proxy-to-presto -upstream=presto:8080 -log-level=TRACE
+
+proxy-minio:
+	consul connect proxy -service=proxy-to-minio -upstream=minio:9090 -log-level=TRACE
+
+connect-to-all:
+	gnome-terminal -- make proxy-redash
```

Comment:
Machine os dependent `gnome-terminal`

Reply:
We can probalby just remove `connect-to-all`. It was just a handy feature for Linux users to automatically open all proxies in separate terminals.

Reply:
I agree this should be done differently, but I'd like to keep it, but create an issue to make a more generic solution. This is very useful for people like Tron Magnus